src/components/Message.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Message (props){
+    const {message} = props
+    return (
+        <div>
```

Comment:
On the lab instructions they give you the exact className that you need to use on each div so your tweets display with the correct format.